indico/htdocs/js/indico/modules/sessions.js

```diff
@@ -0,0 +1,134 @@
+        handleRowSelection();
+        setupTableSorter();
+        setupPersonsAjaxDialog();
+
+        $('#select-all').on('click', function() {
+            $('table.i-table input.select-row').prop('checked', true).trigger('change');
+        });
+
+        $('#select-none').on('click', function() {
+            $('table.i-table input.select-row').prop('checked', false).trigger('change');
+        });
+
+        $('.sessions .toolbar').on('click', '.disabled', function(evt) {
+            evt.preventDefault();
+            evt.stopPropagation();
+        });
+
+        $('#add-new-session').on('click', function() {
+            var $this = $(this);
+            ajaxDialog({
+                url: $this.data('href'),
+                title: $this.data('title'),
+                onClose: function(data) {
+                    if (data) {
+                        $('.sessions-wrapper').html(data.sessions_list);
+                        handleRowSelection();
+                        setupTableSorter();
+                        setupPersonsAjaxDialog();
+                    }
+                }
+            });
+        });
+
+        $('.sessions').on('click', '.edit-session', function() {
+            var $this = $(this);
+            ajaxDialog({
+                url: $this.data('href'),
+                title: $this.data('title'),
+                onClose: function(data) {
+                    if (data) {
+                        $('.sessions-wrapper').html(data.sessions_list);
+                        handleRowSelection();
+                        setupTableSorter();
+                        setupPersonsAjaxDialog();
+                    }
+                }
+            });
+        });
+
+        $('.sessions').on('click', '.js-show-sessions', function() {
+            $(this).closest('tr').nextUntil('tr:not(.session-blocks-row)', 'tr').toggle();
+        });
+
+        $('#remove-selected-sessions').on('indico:confirmed', function(evt) {
+            evt.preventDefault();
+            var $this = $(this);
+            var sessionIds = $('.sessions input:checkbox:checked').map(function() {
+                return $(this).val();
+            }).get();
+
+            $.ajax({
+                url: $this.data('href'),
+                method: $this.data('method'),
+                error: handleAjaxError,
+                traditional: true,
+                data: {session_id: sessionIds},
+                success: function() {
+                    $('.sessions input:checkbox:checked').closest('tr').remove();
+                }
+            });
+        });
+
+        $('.sessions').on('indico:confirmed', '.js-remove-session', function(evt) {
+            evt.preventDefault();
+            var $this = $(this);
+
+            $.ajax({
+                url: $this.data('href'),
+                method: $this.data('method'),
+                headers: {'X-CSRF-Token': $('#csrf-token').attr('content')},
```

Comment:
you don't need to add this, it's added automatically unless you somehow ended up with an absolute URL in `data-href`